HOW_TO_CONTRIBUTE.md

```diff
@@ -9,7 +9,7 @@ Please note we have a [code of conduct](/CODE_OF_CONDUCT.md), please follow it i
 1. Download the dependencies in the developer [requirements file](/requirements/developer.txt).
 2. Add unit tests for features being added or bugs being fixed.
 3. Include any new method/function in the corresponding docs file.
-4. Run `pytest` to verify all unit tests pass.
+4. Run `pytest` to verify all unit tests pass. (If you want to see the overall test coverage, run `pytest --cov=xgi tests` and to see the lines of code not covered by tests, run `pytest  --cov-report term-missing --cov=xgi tests`)
```

Comment:
I'm not sure I would add this here, because it makes the contribution rules look longer and more complicated. Either have a link to another internal page explaining this, or just add a link to pytest's documentation? Also this information can be useful for "advanced" users, but not crucial for first time users I'd say.

Reply:
Good point. I will add a link to pytest-cov!